test/groovy/templates/PiperPipelineStageReleaseTest.groovy

```diff
@@ -158,4 +163,16 @@ class PiperPipelineStageReleaseTest extends BasePiperTest {
 
         assertThat(stepsCalled, hasItem('githubPublishRelease'))
     }
+
+    @Test
+    void testReleaseStageTransportRequestUploadCTS() {
```

Comment:
additional test for unspecified transportRequestUploadCTS (default)

Reply:
Added